public/index.html

```diff
@@ -0,0 +1,215 @@
+      <div id="mission">
+        <h1>Mission Statement</h1>
+        <p id="mission">Here at Origin1, we believe that good coffee should reflect the ground it comes from and the people who tend it. The flavor of a coffee's origin should be the most prominent and the first thing you experience when taking your first sip of any cup of joe. Origin1 seeks to be able to give you the most origin specifc cup by tracking global weather patterns and bringing you the pleasure of tasting the beans from the region having their gold standard of years. We love coffee, and know you will too.</p>
+      </div>
+      <div id="cform">
+        <h1>Contact Us</h1>
+        <p>Feel free to contact us using the form below. We’ll respond to you shortly.</p>
+        <form>
+          <div id="filters">
+            <select name="formTopic">
+              <option value="option" id="option">Select a topic</option>
+              <option value="general" id="general">General Inquiry</option>
+              <option value="donation" id="donation">Donate</option>
+              <option value="complain" id="complain">Complain</option>
+            </select>
+          </div><br>
+          <label>
+            <input type="text" id="fname" class="formInput" placeholder="First Name" maxlength="20"required>
+            <span class="error">Uh oh this field is required</span>
+          </label> <br>
+          <label>
+            <input type="text" id="lname" class="formInput" placeholder="Last Name" maxlength="20" required>
+            <span class="error">Uh oh this field is required</span>
+          </label> <br>
+          <label>
+            <input type="email" id="email" class="formInput" placeholder="Email Address" maxlength="25" required>
+            <span class="error">Uh oh this field is required</span>
+          </label> <br>
+          <label>
+            <input type="tel" id="number" class="formInput" placeholder="Phone Number (optional)" maxlength="15">
+          </label> <br>
+          <label>
+            <textarea class="formInput" id="comment" rows="1"  placeholder="What's on your mind? (optional)" maxlength="250"></textarea>
+          </label> <br>
+          <label> Subscribe to our newsletter
+            <input type="checkbox" id ="subscribe">
+          </label> <br>
+          <div id="submit">
+            <button type="submit" id="submitButton" name="button">Submit</button>
+          </div>
+        </form>
+      </div>
+    </section>
+    <!-- <footer class="tab footer">
+      <h3>I'm the footer</h3>
+    </footer> -->
+    <!-- <section class="loadEl" id="loadSec"></section> -->
+
+    <script src="https://code.jquery.com/jquery-2.2.3.js" integrity="sha256-laXWtGydpwqJ8JA+X9x2miwmaiKhn8tVmOVEigRNtP4=" crossorigin="anonymous"></script>
+    <script src="https://cdn.firebase.com/js/client/2.4.2/firebase.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/2.0.0/handlebars.js"></script>
```

Comment:
I would encourage you to put your CDN's / External libraries up in the head so they've loaded by the time the DOM loads and your other scripts have run.